### PR TITLE
[DOC] Update JMX connector documentation

### PR DESCRIPTION
**Pull request by Akanksha-kedia**
## Description
[DOC] Update JMX connector documentation

## Motivation and Context
To provide more accurate and detailed information to users about the SQL operations supported by the JMX connector in Presto. The current documentation mentions only two schemas (current and history), but the JMX connector actually provides three schemas (current, history, and information_schema). We need to update this information to maintain consistency and accuracy. Also add few examples for information_schema.

Fixes https://github.com/prestodb/presto/issues/22247

## Impact
No impact. Changes in presto-docs(jmx.rst)

## Test Plan
Manually built and tested.
Screenshot 2024-03-20 at 11 01 55 AM
Screenshot 2024-03-20 at 11 02 03 AM
Screenshot 2024-03-20 at 11 02 08 AM

## Contributor checklist

- [x] Please make sure your submission complies with our [development](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#development), [formatting](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#formatting), [commit message](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#commit-formatting-and-pull-requests), and [attribution guidelines](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#attribution).
- [x] PR description addresses the issue accurately and concisely.  If the change is non-trivial, a GitHub Issue is referenced.
- [x] Documented new properties (with its default value), SQL syntax, functions, or other functionality.
- [x] If release notes are required, they follow the [release notes guidelines](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines).
- [x] Adequate tests were added if applicable.
- [x] CI passed.

## Release Notes
Please follow [release notes guidelines](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines) and fill in the release notes below.

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

**Comment by Akanksha-kedia** [comment]
@steveburnett

**Comment by Akanksha-kedia** [comment]
> @steveburnett
please review.

**Comment by Akanksha-kedia** [comment]
Fixes #22247

**Comment by Akanksha-kedia** [comment]
Fixes https://github.com/prestodb/presto/issues/22247

**Comment by Akanksha-kedia** [comment]
@steveburnett @tdcmeehan please review @agrawalreetika

**Review by agrawalreetika — APPROVED** [review]
LGTM

**Comment by steveburnett** [comment]
Please add a release note entry of 
```
== NO RELEASE NOTE ==
```

File: presto-docs/src/main/sphinx/connector/jmx.rst [hunk 1]
```diff
@@ -111,3 +117,13 @@ timestamp column that stores the time at which the snapshot was taken::
      2016-01-28 10:19:00.000 |  21422
      2016-01-28 10:19:10.000 |  31412
     (3 rows)
+
+Information Schema
+^^^^^^^^^^^^^^^^^^
+
+The ``information_schema`` contains metadata about all the other schemas.
+You can see all the available tables in this schema by running the following command:
```
Comment (steveburnett) [inline presto-docs/src/main/sphinx/connector/jmx.rst:125]:
```suggestion
To list all the available tables in this schema, run the following command:
```

File: presto-docs/src/main/sphinx/connector/jmx.rst [hunk 2]
```diff
@@ -47,9 +47,12 @@ Commas in MBean names should be escaped in the following manner:
...
 
-The JMX connector provides two schemas.
+The JMX connector provides three schemas: ``current``, ``history``, and ``information_schema``.
 
-The first one is ``current`` that contains every MBean from every node in the Presto
+Current Schema
+^^^^^^^^^^^^^^
+
+The ``current`` schema contains every MBean from every node in the Presto
 cluster. You can see all of the available MBeans by running ``SHOW TABLES``::
```
Comment (steveburnett) [inline presto-docs/src/main/sphinx/connector/jmx.rst:56]:
```suggestion
cluster. To see all of the available MBeans, runn ``SHOW TABLES``::
```

**Review by steveburnett — CHANGES_REQUESTED** [review]
Thanks. A couple of nits only.

**Comment by Akanksha-kedia** [comment]
@steveburnett @tdcmeehan please review.

File: presto-docs/src/main/sphinx/connector/jmx.rst [hunk 3]
```diff
@@ -47,10 +47,13 @@ Commas in MBean names should be escaped in the following manner:
...
-The JMX connector provides two schemas.
+The JMX connector provides three schemas: ``current``, ``history``, and ``information_schema``.
 
-The first one is ``current`` that contains every MBean from every node in the Presto
-cluster. You can see all of the available MBeans by running ``SHOW TABLES``::
+Current Schema
+^^^^^^^^^^^^^^
+
+The ``current`` schema contains every MBean from every node in the Presto
+cluster. To see all of the available MBeans, runn ``SHOW TABLES``::
```
Comment (steveburnett) [inline presto-docs/src/main/sphinx/connector/jmx.rst:56]:
```suggestion
cluster. To see all of the available MBeans, run ``SHOW TABLES``::
```
I should have caught this last review.

**Review by steveburnett — CHANGES_REQUESTED** [review]
Thanks! One minor spelling nit that I should have caught in my previous review.

**Review by steveburnett — APPROVED** [review]
LGTM! (docs)

Pull updated branch, new local build, everything looks good. Thanks!

**Pull request merged**